### PR TITLE
Show diff on assertionError during compare_jjb_output

### PR DESCRIPTION
**Pull request by erenatas**
Updated `tests/__init__.py` to register `testers/helpers.py`

### CODE REVIEW

File: tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import pytest
+
+pytest.register_assert_rewrite('tests.helpers')
```
Comment (romanek-adam) [inline tests/__init__.py:3]: Shouldn't it be applied just to `tests` instead of `tests.helpers`?

Comment (erenatas) [reply tests/__init__.py:3]: But all I needed to register was the method inside `tests.helpers`, why should I apply it to all of `tests`?

Comment (romanek-adam) [reply tests/__init__.py:3]: Because when `AssertionError` occurs in a test that doesn't use `compare_jjb_output()` the diff would be beneficial too?

Comment (erenatas) [reply tests/__init__.py:3]: Results in the following warning, therefore I have added `--disable-warnings` on tox.

```
============================================================================= warnings summary =============================================================================
tests/__init__.py:3
  /home/erenatas/GitRepos/github/aromanek/jenkins-job-wrecker/tests/__init__.py:3: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: tests
    pytest.register_assert_rewrite('tests')

tests/test_jjb.py::TestJJB::test_ice_setup
tests/test_jjb.py::TestJJB::test_calamari_clients
tests/test_jjb.py::TestJJB::test_non_ascii
  /home/erenatas/GitRepos/github/aromanek/jenkins-job-wrecker/.tox/py36/lib/python3.6/site-packages/jenkins_jobs/cli/subcommand/test.py:55: DeprecationWarning: The 'warn' method is deprecated, use 'warning' instead
    "(Deprecated) The default output behavior of"

-- Docs: https://docs.pytest.org/en/latest/warnings.html
====================================================================== 34 passed, 4 warnings in 0.55s ======================================================================
```

Comment (romanek-adam) [reply tests/__init__.py:3]: Are you sure this was the right thing to do? If the module was already imported then rewriting is not possible. Hiding the warning does not make it work.

The question is - does assertion rewriting work for regular tests without explicitly calling `pytest.register_assert_rewrite()` ?

**Review by romanek-adam — APPROVED** [review]

**Pull request merged**